context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx

```diff
@@ -13,29 +13,26 @@ const useSavedEntitiesSelector = (state) => ({
   removeEntityFromList: state.removeEntityFromList,
 });
 
-function getSavedListsWhichContainEntity(savedLists, savedEntity, entity_type) {
+function getSavedListsWhichContainEntity(savedLists, savedEntity) {
   return Object.entries(savedLists).reduce((acc, [title, obj]) => {
-    if (savedEntity in obj[entity_type]) {
-      return [...acc, title];
-    }
-    return acc;
+    return savedEntity in obj.savedEntities ? [...acc, title] : acc;
```

Comment:
```suggestion
    return (savedEntity in obj.savedEntities) ? [...acc, title] : acc;
```
In case I forget order of operations.

Reply:
Worth a prettier ignore?